### ISSUE TITLE
Allow "nullish" children

### ISSUE THREAD

**Issue by paulobmarcos** [moxystudio/react-carousel#24]
# Issue

While using react-carousel, I found a problem trying to render conditional children:

```js
<Carousel>
    { condition && <div>hello</div> }
    <div>world</div>
</Carousel>
```

I found the problem to be related with the render method of the Carousel:
```js
 {React.Children.map(children, (child, i) => (
     React.cloneElement(child, {
         ...child.props,
         ...
    })
))}
```

We are trying to access the props of a child that is nullish.

# Solution

A solution to this problem would be to use the `toArray` method provided by `React.Children` that will filter out null children.

Like this:
```js
 {React.Children.toArray(children).map((child, i) => (
     React.cloneElement(child, {
         ...child.props,
         ...
    })
))}
```

# Note
I tried to commit the change to a branch but I seem to have no permissions.

Thanks 🙏 
@satazor @jgradim

**Comment by jgradim** [comment]
Nice catch @paulobmarcos! PR is up at https://github.com/moxystudio/react-carousel/pull/25

**Comment by jgradim** [comment]
@paulobmarcos released ` @moxy/react-carousel@0.1.9`
